README.md

```diff
@@ -1,9 +1,79 @@
 # u02-redesign-unit7
 
-The aim of this project is to complete assignment **u02-Redesign** in the course HTML & CSS @ CHAS Academy.
-
+The aim of this project is to complete assignment **u02-Redesign** in the course HTML & CSS @ CHAS Academy.  
 **Due date November 1st 2020.**
 
+## Quick-start
+
+To clone the project and save it locally:
+1. Open your terminal and go to your project/develop folder (where you want to save the clone)
+2. Type in `git clone https://github.com/chas-academy/u02-redesign-unit7.git` and hit enter
```

Comment:
```suggestion
2. Type in `git clone git@github.com:chas-academy/u02-redesign-unit7.git` and hit enter. **NOTE:** This will require the use of SSH key. If you don't have one yet, [follow documentation to set it](https://docs.github.com/en/free-pro-team@latest/github/authenticating-to-github/connecting-to-github-with-ssh)```

Reply:
Yes agreed! I'll try to fix it :)